repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -3185,6 +3184,14 @@ public void updateClassifications(EntityMutationContext context, String guid, Li
             if (updatedRemovePropagations != null && (updatedRemovePropagations != currentRemovePropagations)) {
                 AtlasGraphUtilsV2.setEncodedProperty(classificationVertex, CLASSIFICATION_VERTEX_REMOVE_PROPAGATIONS_KEY, updatedRemovePropagations);
 
+                // If value set true from false and source entity is deleted, remove propagated classification from propagated entities
+                if (updatedRemovePropagations && !currentRemovePropagations) {
```

Comment:
What in case DEFERRED_ACTION_ENABLED is false?
This is not a blocker for us but wanted to know what will happen

Reply:
Basically this means we will perform the propagation in the request itself, not as background task.